### DISCUSSION TITLE
Just another feature

### DISCUSSION THREAD

**Discussion by kavi6024** [community/community#8596] [Copilot Conversations]
Can Github Copilot refactor a code to reduce time and space complexity ? Or can Github Copilot change a code from one language to another ? Or can Github Copilot write code according to the files in my current project ? Or can Github Copilot center div from memory (LOL) ? Or can Github Copilot solve new problems created by user using Input-Output statements given ? These are some features that coder enthusiasts like me expect from Github Copilot. But seriously, great job well done, team Github.

Expecting a Copilot upgrade,
#CodeLover

**Comment by SmallLemur** [comment]
Unfortunately Copilot cannot do all of this at the moment, but we will definitely keep improving the model and its functionalities over time :)

**Comment by kavi6024** [comment]
Ok. And really, you guys are doing a great job.

**Reply by SmallLemur** [reply]
Thank you so much !
